templates/solutions/telco/5g-core.html

```diff
@@ -0,0 +1,595 @@
+              </div>
+              <div class="col-4">
+                <ul class="p-list--divided">
+                  <li class="p-list__item is-ticked">Enhanced Platform Awareness (EPA)</li>
+                  <li class="p-list__item is-ticked">Service Chaining and Integration</li>
+                  <li class="p-list__item is-ticked">Life Cycle Management</li>
+                  <li class="p-list__item is-ticked">Scalability and Flexibility</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <p>
+              <a href="/openstack">Read more about Canonical OpenStack&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <hr class="p-rule--highlight" />
+        <div class="p-card--overlay u-no-padding--top" style="padding: 1.5rem;">
+          <div class="p-section--shallow">
+            <div class="row">
+              <div class="p-section--shallow">
+                <h3>Orchestrate your Core with Kubernetes</h3>
+              </div>
+              <p class="u-text--muted">
+                As a proud member of the Cloud Native Computing Foundation (CNCF), Canonical uses Kubernetes to manage Containerised Network Functions (CNFs) in telecom networks. Kubernetes is the leading container orchestration platform and automates deployment, scaling, and operations.
+              </p>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="p-section--shallow">
+            <div class="row">
+              <div class="col-2">
+                <h4 class="p-text--small-caps">What's included</h4>
+              </div>
+              <div class="col-4">
+                <ul class="p-list--divided">
+                  <li class="p-list__item is-ticked">Cloud native</li>
+                  <li class="p-list__item is-ticked">Automated scaling</li>
+                  <li class="p-list__item is-ticked">Self-healing</li>
+                  <li class="p-list__item is-ticked">Rolling updates</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <p>
+              <a href="/engage/hpe-telco-validated-design-with-canonical-kubernetes">Discover how HPE and Canonical's Kubernetes Telco validated design can future-proof your network&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Canonical's 5G Core architecture</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-container">
+          {{ image(url="https://assets.ubuntu.com/v1/9d96ea3a-5gcore.png",
+                    alt="5G Core architecture",
```

Comment:
@Sophie-32 is it possible to get more descriptive alt text here? This isn't just a decorative image, from how this reads this diagram is supposed to communicate the core architecture. But non-sighted users wouldn't get any of this info as is, they'll just hear "5G core architecture". Also, this is illegible for sighted users as well on smaller screens. I think a longer description in addition to the general alt text would be useful here, similar to what we did for a previous diagram (will try to find the link for reference)